app/Exceptions/Handler.php

```diff
@@ -4,13 +4,15 @@
 
 use Illuminate\Foundation\Exceptions\Handler as ExceptionHandler;
 use App\Helpers\Helper;
+use Illuminate\Support\Facades\RateLimiter;
```

Comment:
A little weird that these got added when none of the code we added (or removed) necessarily adds any new dependencies? Probably not a big deal.